Taskfile.yml

```diff
@@ -29,39 +33,132 @@ tasks:
       - mkdir -p {{.LOCATION}}
       - mv {{.BINARY}} {{.LOCATION}}/{{.BINARY}}
 
+  ci:
+    desc: Workflow to run in CI
+    cmds:
+      - task: go:setup
+      - task: go:has-latest-opslevel-go
+      - task: lint
+      - task: test
+
+  clean:
+    desc: Clean up terraform files from "{{.WORKSPACE_DIR}}"
+    dir: "{{.WORKSPACE_DIR}}"
+    prompt: Remove '*.tfstate .terraform.lock.hcl ./terraform.* .terraform/**' from '{{.WORKSPACE_DIR}}' directory?
+    cmds:
+      - cmd: rm -rf *.tfstate .terraform.lock.hcl ./terraform.* .terraform/
+        ignore_error: true
+
+  destroy:
+    desc: Run "terraform destroy" in "{{.WORKSPACE_DIR}}"
+    cmds:
+      - task: terraform-command
+        vars: { TF_COMMAND: "destroy", TF_CMD_DIR: "{{.WORKSPACE_DIR}}" }
+
+  fix:
+    desc: Fix formatting and linting
+    cmds:
+      - task: go:fix
+      - task: format-fix
```

Comment:
format-fix should not be part of linting since the data in workspace/ doesn't matter - it's only used for tophatting.